sources/views/customersData.js

```diff
@@ -0,0 +1,49 @@
+import { JetView } from "webix-jet";
+
+export default class CustomersData extends JetView {
+	constructor(app, name, data) {
+		super(app, name);
+		this._componentData = data;
+	}
+	config() {
+		return {
+			rows: [
+				{
+					view: "datatable",
+					autoConfig: true,
+					localId: "table",
+					editable: true,
+					select: true,
+					editaction: "dblclick",
+					scroll: "y",
+				},
+				{
+					cols: [
+						{},
+						{
+							view: "button", value: "Add", width: 100, css: "webix_primary", click: () => {
+								const item = { Name: "New Item" };
```

Comment:
prefer setting class methods instead of performing logic in event handlers. and, you can store frequently uses view as a class prop:

~~~
init(){
   this.table = this.$$("table"); //get view by local id
}
deleteItem(){
         //.. get selected id .. 
         this.table.remove(sel);
}
~~~

Reply:
Ok, rewrote them)

Reply:
now you have queryView in init and `this.$$` in other methods, but it is better to locate the datatable once  in `init` as shown above

Reply:
Now I got it, thank you.